features/resize.yml

```
@@ -0,0 +1,11 @@
+name: Resize
```

This is a tough one. I think there's a risk of confusion with the `resize` method on resizable `ArrayBuffer`s, so we ought to do a relatively rare thing and use parenthetical disambiguation text:

```suggestion
name: resize (CSS property)
```